test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/007_VD_parameter_in_API_DB_app_version_less_then_parameter_version.lua

```diff
@@ -0,0 +1,34 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: The app is able to receive the parameters from HMI in case app version is less then parameters version,
+-- parameter is listed in DB and API
+-- In case:
+-- 1) App is registered with syncMsgVersion=5.0
+-- 2) New params in `FuelRange` structure have since=6.0 and DB
```

Comment:
This script is not in scope for new proposal. FuelRange is old parameter, versioning should be checked only for new and deprecated parameters.